docs/introduction/Sidechain-SDK-extension.rst

```diff
@@ -79,33 +79,33 @@ Custom box creation
 
   a) SDK Box extension Overview
 
-To build a real application, a developer will need more to do more than receive, transfer, and send coins back. A distributed app, built on a sidechain, will typically have to define some custom data that the sidechain users will be able to exchange according to a defined logic. The creation of new Boxes requires the definition of four new classes. We will use the name Custom Box as a definition for some abstract custom Box:  
+To build a real application, a developer will need to do more than receive, transfer, and send coins back. A distributed app, built on a sidechain, will typically have to define some custom data that the sidechain users will be able to exchange according to a defined logic. The creation of new Boxes requires the definition of four new classes. We will use the name Custom Box as a definition for some abstract custom Box:  
 
 
 +--------------------------------------+------------------------------------------------------------------------------------+
 | Class type                           | Class description                                                                  |
 +======================================+====================================================================================+
 | Custom Box Data class                | -- Contains all custom data definitions plus proposition for Box                   |
-|                                      | -- Provide required information for serialization of Box Data                      |
-|                                      | -- Define the way for creation new Custom Box from current Custom Box Data         |
+|                                      | -- Provides required information for serialization of Box Data                     |
+|                                      | -- Defines the way to create a new Custom Box from current Custom Box Data         |
 +--------------------------------------+------------------------------------------------------------------------------------+
-| Custom Box Data Serializer Singleton | -- Define the way how to parse bytes from Reader into Custom Box Data object       |
-|                                      | -- Define the way how to put boxData object into Writer Parsing function used in a |
+| Custom Box Data Serializer Singleton | -- Defines how to parse bytes from Reader into Custom Box Data object              |
+|                                      | -- Defines the way to put boxData object into Writer Parsing function used in a    |
```

Comment:
@zennermac - Defines how to put the BoxData object into the Writer parsing function. Used in a Serializer class, but can be put in that class as well. However, it can be defined elsewhere.